modules/nf-core/bcftools/annotate/main.nf

```diff
@@ -8,29 +8,33 @@ process BCFTOOLS_ANNOTATE {
         'quay.io/biocontainers/bcftools:1.16--hfe4b78e_1' }"
 
     input:
-    tuple val(meta), path(input)
+    tuple val(meta), path(input), path(index), path(annotations), path(annotations_index)
+    path(header_lines)
 
     output:
-    tuple val(meta), path("*_annotated.vcf.gz"), optional:true , emit: vcf
-    tuple val(meta), path("*_annotated.bcf")   , optional:true , emit: bcf
-    path "versions.yml"                                        , emit: versions
+    tuple val(meta), path("*.{vcf,vcf.gz,bcf,bcf.gz}"), emit: vcf
+    path "versions.yml"                               , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
-    def args = task.ext.args ?: ''
-    def prefix = task.ext.prefix ?: "${meta.id}"
-
```

Comment:
Can you also add a file name collision check? (see [here](https://nf-co.re/docs/contributing/modules#naming-conventions) part 6 for an example)

Reply:
Nice! Added 👍🏻